src/app/home/home.component.html

```diff
@@ -1,13 +1,292 @@
 ﻿<div class="container-fluid">
     <div class="welcomeuser">
-    <div class="row col-md-12">
-        <div class="col-md-6 text">
-            <h1>Welcome {{currentUser.firstName}}!</h1>
-            <h2>You are logged in as the {{currentUser.role}}!!</h2>
+        <div class="row col-md-12">
+            <div class="col-md-6 text">
+                <h1>Welcome {{currentUser.firstName}}!</h1>
+                <h2>You are logged in as the {{currentUser.role}}!!</h2>
+            </div>
+            <div class="col-md-6">
+                <img src="assets/images/cover.png">
+            </div>
         </div>
-        <div class="col-md-6">
-            <img src="assets/images/cover.png">
+    </div>
+    <div class="content">
+        <!-- USER 1 : Can view courses and features only. -->
+        <div *ngIf="currentUser.role === 'user1'">
```

Comment:
Instead of having the html of all cases in a single file, can we have them written in different files and render it based on the condition. Its fine if you leave this as it is.

Reply:
Making templates for each condition doesn't seem to be useful as these will not be used anywhere else but only on this page.